bin/main.rb

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env ruby
+puts 'Hello world!'
```

Comment:
Please make this file executable as suggested.

Reply:
We changed the permissions